sass/_otwsu.scss

```
@@ -1,7 +1,26 @@
-.podcasts-grid {
-    display: grid;
-    grid-template-columns: 2fr 2fr;
-    gap: 20px;
+.podcasts-content {
+    display: flex;
+    @media (max-width: 767px) {
+        flex-direction: column;
+    }
+    max-width: 1000px;
+    margin: 0 auto 2em auto;
```

```suggestion
    margin: 0 auto 2em;
```
Has the same meaning.